src/test/java/io/stargate/sgv2/jsonapi/api/v1/tables/FindPaginationTableIntegrationTest.java

```diff
@@ -31,11 +38,86 @@ public final void dropScenario() {
 
   @Test
   public void findWithPageState() {
+    // `find_all` command to get the next page state
```

Comment:
also add a test that goes through all the pages in a partition, checks the rows have the clustering key values we expect, in the order we expect, and the last page is a  partial page